src/lib/components/carrousel.svelte

```diff
@@ -58,19 +57,20 @@
   .carrousel {
     position: relative;
     width: 100%;
-    aspect-ratio: 16/7;
+    height: 100%;
   }
-  .image {
+  .image-container {
     z-index: -1;
     position: absolute;
-    bottom: 50px;
-    aspect-ratio: 16/7;
     width: 100%;
+    height: 100%;
+    display: flex;
   }
   img {
     object-fit: cover;
     height: 100%;
     width: 100%;
+    padding-bottom: 50px;
```

Comment:
this is still really dodgy

Reply:
It is the space of the round buttons without this space they are stuck to the image I change it to the parent of the image

Reply:
the container has a height now with flex 1

Reply:
the buttons are absolutely with a botton 0 if I don't give it that margin they are positioned above the image at the end of the container it doesn't matter the flex 1

Reply:
this should not be on the image, it should be on the relative container - that way it makes sense